### ISSUE TITLE
Add Bloom Filter metrics

### ISSUE THREAD

**Issue by tzach** [scylladb/scylladb#1066]
Cassandra support the following Bloom filter metrics:
- Bloom Filter False Positives 
- Bloom Filter False Ratio 
- Bloom Filter Space Used
- Bloom Filter Disk Space Used

Scylla already expose this metrics via cfstat, it will be useful to also share them via collectd metrics.

**Comment by benoit-canet** [comment]
@tzach: Are you sure you want them global like in cfstat ?

**Comment by tzach** [comment]
@benoit-canet at what level do Cassandra have these metrics?

**Comment by benoit-canet** [comment]
@tzach: I looked into scylla: we seems to do them both a per cf level by providing a cf name and per database level

**Comment by benoit-canet** [comment]
@Tzach: https://github.com/apache/cassandra/blob/173dbab24b9acba46279ca3477f080e57f52636c/src/java/org/apache/cassandra/metrics/KeyspaceMetrics.java#L219

Look its per key space

**Comment by tzach** [comment]
@benoit-canet make sense for Scylla as well

**Comment by benoit-canet** [comment]
@tzach: ok I am coding it.

**Comment by benoit-canet** [comment]
@tzach: I don't see how we could fit a ratio (floating point) into the seastar metrics api

**Comment by gleb-cloudius** [comment]
On Wed, Jun 28, 2017 at 06:05:35AM -0700, Benoît Canet wrote:
> @tzach: I don't see how we could fit a ratio (floating point) into the seastar metrics api 
> 
What's the problem? It supports floating point directly.

--
			Gleb.

**Comment by benoit-canet** [comment]
ok

2017-06-28 15:26 GMT+02:00 Gleb Natapov <notifications@github.com>:

**Comment by benoit-canet** [comment]
@tzach: It look difficult to get back the column familly from the shema_ptr structure inside schema.
I will keep looking for a way to do this.

**Comment by benoit-canet** [comment]
@tzach: don't worry I found my way through it.

**Comment by benoit-canet** [comment]
@tzach: Hi, I have a patch on the list which is just missing the false ratio.

Can we live without this metrics or do you absolutly want it ?

**Comment by tzach** [comment]
If the patch covers the rest of the metric in this issue its already an improvement.
Once it is merged, please open a new issue just for the false ratio.

**Comment by tzach** [comment]
No progress and no urgency. Closing.
